### PR TITLE
fix: back to tcp

### PR DESCRIPTION
**Pull request by JoanFM**
May be the reason for https://github.com/jina-ai/jina-hub/issues/472?

### CODE REVIEW

**Comment by theUnkownName** [comment]
That `publish_all_port` has nothing to do right?

**Comment by JoanFM** [comment]
> That `publish_all_port` has nothing to do right?
should not

**Review by theUnkownName — APPROVED** [review]

**Review by rutujasurve94 — APPROVED** [review]

File: jina/peapods/container.py [hunk 1]
```diff
@@ -69,13 +69,14 @@ def post_init(self):
         _args = kwargs2list(non_defaults)
         self._container = self._client.containers.run(self.args.uses, _args,
                                                       detach=True, auto_remove=True,
-                                                      ports={f'{v}': v for v in
+                                                      ports={'%d/tcp' % v: v for v in
```
Comment (maximilianwerk) [inline jina/peapods/container.py:72]: Please stick with f-strings: `f'{v}/tcp'.

Comment (JoanFM) [reply jina/peapods/container.py:72]: yes, my fault, it was just a conservative revert

**Review by hanxiao — APPROVED** [review]

**Pull request merged**